ScreenToGif/Resources/Localization/StringResources.uk.xaml

```diff
@@ -13,28 +13,28 @@
     <s:String x:Key="S.Cancel">Скасувати</s:String>
     <s:String x:Key="S.Yes">Так</s:String>
     <s:String x:Key="S.No">Ні</s:String>
-    <!--<s:String x:Key="S.Add">Add</s:String>-->
-    <!--<s:String x:Key="S.Edit">Edit</s:String>-->
+    <s:String x:Key="S.Add">Додати</s:String>
+    <s:String x:Key="S.Edit">Редагувати</s:String>
     <!--<s:String x:Key="S.Id">Id</s:String>-->
-    <!--<s:String x:Key="S.Title">Title</s:String>-->
-    <!--<s:String x:Key="S.Description">Description</s:String>-->
-    <!--<s:String x:Key="S.SelectColor">Click here to select the color.</s:String>-->
+    <s:String x:Key="S.Title">Назва</s:String>
+    <s:String x:Key="S.Description">Опис</s:String>
+    <s:String x:Key="S.SelectColor">Натисніть, щоб вибрати колір.</s:String>
     <s:String x:Key="S.Suppress">Припинити</s:String>
     <s:String x:Key="S.Preview">Попередній перегляд</s:String>
     <s:String x:Key="S.Size">Розмір</s:String>
     <s:String x:Key="S.Background">Тло</s:String>
     <s:String x:Key="S.Color">Колір:</s:String>
     <s:String x:Key="S.Delay">Затримка</s:String>
     <s:String x:Key="S.DelayMs">Затримка (мс):</s:String>
-    <!--<s:String x:Key="S.ValueMs">Value -+:</s:String>-->
+    <s:String x:Key="s.valuems">Значення -+:</s:String>
```

Comment:
Don't change the ID of the text resource (names are case sensitive).
The app won't be able to find the text if you do that. 

S.ValueMs